pages/1.12/cli/index.md

```diff
@@ -8,137 +8,63 @@ excerpt: Understanding the command line interface utility in DC/OS
-
-# Environment variables
-
-The DC/OS CLI supports the following environment variables, which can be set dynamically.
-
-<a name="dcos-cluster"></a>
-#### `DCOS_CLUSTER` (DC/OS CLI 0.5.x and higher only)
-
-The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the attached cluster, set the variable with the command:
-
-```bash
-export DCOS_CLUSTER=<cluster_name>
-```
-
-<a name="dcos-config"></a>
-#### `DCOS_CONFIG` (DC/OS CLI 0.4.x only)
-
-The path to a DC/OS configuration file. If you put the DC/OS configuration file in `/home/jdoe/config/dcos.toml`, set the variable with the command:
+In order to interact with your cluster, you first need to set up the CLI.
 
 ```bash
-export DCOS_CONFIG=/home/jdoe/config/dcos.toml
+$ dcos cluster setup <cluster-url>
 ```
 
-If you have the `DCOS_CONFIG` environment variable configured:
+After following the login flow, your CLI is now ready to interact with your cluster. You will notice that it now has additional commands such as `marathon`, `node`, `package` etc. These commands come from the [plugins](/1.12/cli/plugin), dcos-core-cli and, if applicable, dcos-enterprise-cli, which get automatically installed as part of the setup command.
 
-- After conversion to the [new configuration structure](#configuration-files), `DCOS_CONFIG` is no longer honored.
-- Before you call `dcos cluster setup`, you can change the configuration pointed to by `DCOS_CONFIG` using `dcos config set`. This command prints a warning message saying the command is deprecated and recommends using `dcos cluster setup`.
+# Listing your clusters
 
-
-<a name="dcos-dir"></a>
-#### `DCOS_DIR` (DC/OS CLI 0.5.x and higher only)
-
-The path to a DC/OS configuration directory. If you want the DC/OS configuration directory to be `/home/jdoe/config`, set the variable with the command:
-
-```bash
-export DCOS_DIR=/home/jdoe/config
-```
-
-1. Optionally set `DCOS_DIR` and run `dcos cluster setup` command.
-
-    ```
-    export DCOS_DIR=<path/to/config_dir> (optional, default when not set is ~/.dcos)
-    dcos cluster setup <url>
-    ```
-
-   This setting generates and updates per cluster configuration under `$DCOS_DIR/clusters/<cluster_id>`. Sets newly set up cluster as the attached one.
-
-
-<a name="dcos-ssl-verify"></a>
-#### `DCOS_SSL_VERIFY`
-Indicates whether to verify SSL certificates or set the path to the SSL certificates. You must set this variable manually. Setting this environment variable is equivalent to setting the `dcos config set core.ssl_verify` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+The DC/OS CLI can work with multiple clusters, the one you just configured should now appear with the following command:
```

Comment:
Rewrite "The DC/OS CLI can work with multiple clusters, the one you just configured should now appear with the following command:"

as 
The DC/OS CLI can work with multiple clusters, the latest configured cluster should now appear with the following command: